### DISCUSSION TITLE
Can Draft 3 Version of JSON schema be used for JSON MRFs?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#378] [Q&A] [answered]
Are there any restrictions for version of JSON?  Can Draft 3 Version of JSON be used?

**Comment by shaselton-usds** [comment, marked as answer]
I don't think Draft 3 supports some of the features being leveraged in the current schema. Mainly key attributes such as `anyOf` and `oneOf`
